beamer/state_machine.py

```diff
@@ -66,9 +66,9 @@ class Context:
 
 def process_event(event: Event, context: Context) -> HandlerResult:
     if isinstance(event, SourceChainEvent) and event.chain_id != context.source_chain_id:
-        return True, []
+        return True, None
     if isinstance(event, TargetChainEvent) and event.chain_id != context.target_chain_id:
-        return True, []
+        return True, None
```

Comment:
Nitpick: could you please move this change into the previous commit, where the list was introduced? Seems like that commit should have this fix which is anyway not all that related to other changes in this commit.